test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerIntegrationTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.DotNet.PlatformAbstractions;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
+{
+    public class BuildServerIntegrationTest : MSBuildIntegrationTestBase, IClassFixture<BuildServerTestFixture>
+    {
+        private readonly string _pipeName;
+
+        public BuildServerIntegrationTest(BuildServerTestFixture fixture)
+        {
+            _pipeName = fixture.PipeName;
+        }
+
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public Task Build_SimpleMvc_WithServer_UsingDotnetMSBuild_CanBuildSuccessfully()
+            => Build_SimpleMvc_CanBuildSuccessfully(MSBuildProcessKind.Dotnet);
+
+        private async Task Build_SimpleMvc_CanBuildSuccessfully(MSBuildProcessKind msBuildProcessKind)
+        {
+            var result = await DotnetMSBuild(
+                "Build", 
+                $"/p:RazorCompileOnBuild=true /p:UseRazorBuildServer=true /p:RazorBuildServerPipeName={_pipeName}",
+                msBuildProcessKind: msBuildProcessKind);
+
+            Assert.BuildPassed(result);
+            Assert.FileExists(result, OutputPath, "SimpleMvc.dll");
+            Assert.FileExists(result, OutputPath, "SimpleMvc.pdb");
+            Assert.FileExists(result, OutputPath, "SimpleMvc.PrecompiledViews.dll");
+            Assert.FileExists(result, OutputPath, "SimpleMvc.PrecompiledViews.pdb");
+
+            if (RuntimeEnvironment.OperatingSystemPlatform != Platform.Darwin)
```

Comment:
You can simplify this verification, we don't need to test everything everywhere. It's probably a good simplification to remove the check for the console output